host/linux/time.c

```diff
@@ -28,3 +29,8 @@ void oe_handle_get_time(uint64_t arg_in, uint64_t* arg_out)
     if (arg_out)
         *arg_out = _time();
 }
+
+int oe_localtime(time_t* timep, struct tm* result)
+{
+    return !localtime_r((const time_t*)timep, result);
```

Comment:
Why is the cast needed here?   You should be able to pass a non-const ptr to a const argument without a cast.

Reply:
Fixed, thanks!